tsconfig.json

```diff
@@ -12,13 +13,13 @@
 		"noImplicitReturns": false,
 		"noUnusedParameters": true,
 		"noUncheckedIndexedAccess": true,
-		"resolveJsonModule": true,
+		"resolveJsonModule": false,
 		"esModuleInterop": true,
 		"skipLibCheck": false,
 		"paths": {
-			"*": ["./node_modules/@types/*", "./types/*"]
+			"stylelint": ["./types/stylelint/index.d.ts"]
 		}
 	},
-	"include": ["lib", "types", "package.json"],
-	"exclude": ["**/*.test.{js,mjs}", "**/__tests__/", "lib/**/*.cjs"]
+	"include": ["lib/**/*.mjs", "types/**/*.ts"],
+	"exclude": ["**/__tests__/**"]
```

Comment:
question

The removal of `"**/*.test.{js,mjs}"` didn't have any consequence?
What's special about `"**/__tests__/**"` that warrants to keep excluding it?

Reply:
> The removal of `"**/*.test.{js,mjs}"` didn't have any consequence?

No consequence. Because all `*.test.{js,mjs}` files are under the `__tests__` directory:

```console
$ git ls-files lib/ | grep '\.test\.' | grep -v '__tests__'
(no output)
```

> What's special about `"**/__tests__/**"` that warrants to keep excluding it?

This exclusion aims to prevent so many type-check errors in the test files.